CMakeLists.txt

```diff
@@ -176,6 +178,14 @@ if(CELERITY_USE_MIMALLOC)
   fetch_content_from_submodule(mimalloc vendor/mimalloc)
 endif()
 
+if(CELERITY_TRACY_SUPPORT)
+  set(TRACY_FIBERS ON CACHE BOOL "" FORCE)
+  set(TRACY_STATIC ON CACHE BOOL "" FORCE)
+  set(TRACY_NO_SAMPLING ON CACHE BOOL "" FORCE)
+  set(TRACY_NO_SYSTEM_TRACING ON CACHE BOOL "" FORCE)
```

Comment:
Do we want to FORCE these? The user might want them?

Reply:
AFAIK `FORCE` is needed to override the default that Tracy passes to these `option()`s.